tutorials/btp-integration-suite-execute-batch-payload/btp-integration-suite-execute-batch-payload.md

```diff
@@ -0,0 +1,377 @@
+    import com.sap.gateway.ip.core.customdev.util.Message;
+    import java.util.HashMap;
+    import java.util.Arrays;
+
+    //Insert script
+    def Message processData(Message message) {
+
+        //Headers
+        List paramList = new ArrayList();
+        paramList.add(Arrays.asList(108, 'test', 2021-01-01,1,1,'test@gmail.com'));
+        paramList.add(Arrays.asList(107, 'test', 2021-01-01,1,1,'test@gmail.com'));
+        paramList.add(Arrays.asList(111, 'test', 2021-01-01,1,1,'test@gmail.com'));
+        paramList.add(Arrays.asList(105, 'test', 2021-01-01,1,1,'test@gmail.com'));
+        message.setHeader("CamelSqlParameters",paramList);
+
+        //Body
+        message.setBody("INSERT INTO testjdbcsample (emp_id,emp_name,join_date,a1,a2,email) VALUES(?,?,?,?,?,?)");
+        return message;
+    }
+    ```
+3. Select the **Batch Mode** checkbox and click **Deploy**.
+
+4. Run a blank query in Postman to send the payload which we have entered via Groovy Script and establish a connection between Postman and your integration flow.
+    <!-- border -->![Send Payload from Postman for Groovy Script](Postman-Groovy.png)
+
+5. Let's verify if the payload has been successfully sent to the database. Remove **Groovy Script**, deselect the **Batch Mode** and **Deploy** the integration flow. Then, run the following query from Postman:
+`select * from testjdbcsample`
+    <!-- border -->![Link text e.g., Destination screen](Postman-Groovy-Success.png)
+
+This displays the table data. Hence, we have successfully sent a payload via Groovy Script. You can now alter the table data and verify the Atomic and Non Atomic behaviours as we did earlier.
+
+### Atomic Batch Operation with Groovy Script
+We will replicate the above Atomic scenario with a Groovy Script.
+
+
+1. Choose **Atomic Batch Mode** and add the following script in Groovy Script as explained in Step 8.
+
+    Here we have modified the third and fourth record of the original Groovy Script.
+
+    ```
+        import com.sap.gateway.ip.core.customdev.util.Message;
+        import java.util.HashMap;
+        import java.util.Arrays;
+
+        //Insert script
+        def Message processData(Message message) {
+
+            //Headers
+            List paramList = new ArrayList();
+            paramList.add(Arrays.asList(108, 'test', 2021-01-01,1,1,'test@gmail.com'));
+            paramList.add(Arrays.asList(107, 'test', 2021-01-01,1,1,'test@gmail.com'));
+            paramList.add(Arrays.asList(112, 'test112', 2021-01-01,1,1,'test@gmail.com'));
+            paramList.add(Arrays.asList(106, 'test106', 2021-01-01,1,1,'test@gmail.com'));
+            message.setHeader("CamelSqlParameters",paramList);
+
+            //Body
+            message.setBody("INSERT INTO testjdbcsample (emp_id,emp_name,join_date,a1,a2,email) VALUES(?,?,?,?,?,?)");
+            return message;
+        }
+    ```  
+2. **Send** a blank query in Postman to send the payload which we have entered via Groovy Script.
```

Comment:
Don't say "we" when addressing the author. Say "you" instead!
Missing comma before "which"
Check upper/lower case-> "Script"

Reply:
done